model-server/src/main/kotlin/org/modelix/model/server/CmdLineArgs.kt

```diff
@@ -33,6 +33,14 @@
     @Parameter(names = ["-port", "--port"], description = "Set port", converter = IntegerConverter::class)
     var port: Int? = null
 
+    // 10 seconds was the default value in NettyApplicationEngine when this option was implemented.
+    @Parameter(
+        names = ["-response-write-timeout-seconds", "--response-write-timeout-seconds"],
+        description = "Timeout in seconds for sending responses to client. Values smaller or equal to 0 disable the timeout. Defaults to 10 seconds if unset.",
+        converter = IntegerConverter::class,
+    )
+    var responseWriteTimeoutSeconds: Int = 10
```

Comment:
Could be moved into a `DEFAULT_RESPONSE_WRITE_TIMEOUT_SECONDS` const val if we want to avoid this warning. Otherwise, can you add a [suppression](https://detekt.dev/docs/introduction/suppressing-rules/)?

Reply:
A cool thing with a `const` is that it can be used it in the annotation string.
```
description = "Timeout in seconds for sending responses to clients. Values smaller or equal to 0 disable the timeout. Defaults to $DEFAULT_RESPONSE_WRITE_TIMEOUT_SECONDS seconds if unset."
```